src/libbson/tests/json/bson_corpus/document.json

```diff
@@ -17,6 +17,26 @@
             "description": "Single-character key subdoc",
             "canonical_bson": "160000000378000E0000000261000200000062000000",
             "canonical_extjson": "{\"x\" : {\"a\" : \"b\"}}"
+        },
+        {
+            "description": "Dollar-prefixed key in sub-document",
```

Comment:
These test changes came from https://github.com/mongodb/specifications/commit/851ca10388cced05117ceace0ced4b7e1aac02c4 for [DRIVERS-1237](https://jira.mongodb.org/browse/DRIVERS-1237), which would have been addressed by [CDRIVER-3895](https://jira.mongodb.org/browse/CDRIVER-3895).